<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardConf</class>
 <widget class="QWidget" name="BoardConf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Game Board</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <property name="topMargin">
       <number>26</number>
      </property>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Height:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="sbHeight">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Depth:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="sbDepth">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="chkBoard3d">
        <property name="text">
         <string>3D</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="sbWidth">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Width:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>chkBoard3d</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbDepth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>220</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>board3dToggled(bool)</slot>
 </slots>
</ui>
